itl/command-plugins-windows.conf

```diff
@@ -317,3 +317,31 @@ object CheckCommand "users-windows" {
 		}
 	}
 }
+
+object CheckCommand "file-age-windows" {
+        command = [ PluginDir + "/check_file_age.cmd" ]
```

Comment:
The `.cmd` file is just a wrapper for the Powershell script, is there a reason for this? Can't we call directly the Powershell script?

Reply:
Hi!
Thank you for the feedback. I will move the definition to plugins-contrib.d/windows.conf file. The reason why I used a wrapper script. I was not able to call the PowerShell script directly. Maybe we need this functionality in the future, however I found this way to use the script in Icinga2 for Windows.